### PR TITLE
Added a new tutorial section on upgrading from open source to Enterpr…

### LINKED ISSUE #441: It would help to have doc on how to migrate from OSS to Ent. Edition
**Describe the bug**
It would help to have doc on how to migrate from OSS to Ent. Edition. I only know the method of exporting each/all the projects and importing them.

**Source page**
docs.rundeck.com

### PR DESCRIPTION
**Pull request by chrismcg14**
…ise. Added it to the nav bar and side bars. Created a main page that describes and links the two different methods (on the same server and on a new server). Created said pages for the two methods. fixes #441

### CODE REVIEW

File: docs/tutorials/oss-to-enterprise-migration-new.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+```
+        export RDECK_BASE=$HOME/rundeck; # or where you like it
+```
+* Create the directory for the installation:
+```
+        mkdir -p $RDECK_BASE
+```
+* Copy the executable war to the installation directory:
+```
+        cp rundeck-3.3.2-20200817.war $RDECK_BASE
```
Comment (fdevans) [inline docs/tutorials/oss-to-enterprise-migration-new.md]: this line also needs {{{rundeckVersionFull}}} right?

File: docs/tutorials/oss-to-enterprise-migration-same.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+```
+        export RDECK_BASE=$HOME/rundeck; # or where you like it
+```
+* Create the directory for the installation:
+```
+        mkdir -p $RDECK_BASE
+```
+* Copy the executable war to the installation directory:
+```
+        cp rundeck-3.3.2-20200817.war $RDECK_BASE
```
Comment (fdevans) [inline docs/tutorials/oss-to-enterprise-migration-same.md]: {{{rundeckVersionFull}}} ?

File: docs/tutorials/oss-to-enterprise-migration-new.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
+# Upgrading from Open Source to Enteprise Using a New Server
+
+Congratulations, you have decided to use Rundeck Enterpise! You are on you way to automating your workflows. Before that happens, you need to get your Rundeck Enterprise set up. Fear not, this guide will get you set up no matter what kind of Rundeck user you are. 
+
+## RPM Upgrade
+
+* Delete job execution history. If you do not, then the exporting of projects will take a much longer time.
```
Comment (fdevans) [inline docs/tutorials/oss-to-enterprise-migration-new.md]: "Delete unnecessary job execution history.  By trimming this history exporting the jobs will proceed much faster and have you up and running sooner."

Language needs to be suggestion here and provide the positive reason why versus the negative.

File: docs/tutorials/oss-to-enterprise-migration-new.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+* Export project setting from Rundeck Open Source. Go to the project that you wish to export, go to project settings and select "Export Archive." On the next page, you will be able to choose what all you want exported (Jobs, Executions, Configurations, etc.)
```
Comment (fdevans) [inline docs/tutorials/oss-to-enterprise-migration-new.md:11]: Isn't there a link or page about exporting projects out there?

File: docs/tutorials/oss-to-enterprise-migration-new.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+        service rundeckd restart
+```
+* Once in Rundeck, import the project archive files that you created in step 2. Create a new project, go to project settings and select "Import Archive." On the next page, you will be able to choose what all you want imported (Jobs, Executions, Configurations, etc.)
+* Install license by choosing the license key file that was sent to you by a Rundeck team member. 
+* Set execution mode to active
+* Go to cluster manager, select  jobs, and select all orphan jobs and click action button/Change Job Schedule Owner and assign to a cluster member.
+
+## Debian/Ubuntu Upgrade
+
+* Delete job execution history. If you do not, then the exporting of projects will take a much longer time.
```
Comment (fdevans) [inline docs/tutorials/oss-to-enterprise-migration-new.md]: These are repeated.  We might want a section at the top for "Pre-Requisites" or "Prepare your upgrade" so we only have to manage that content once.  DRY principles ;)

**Comment by fdevans** [comment]
Why are there two files call "-new" and "-same"?

File: docs/tutorials/oss-to-enterprise-migration-new.md (hunk 1 above)
Comment (jplassnibatt) [reply docs/tutorials/oss-to-enterprise-migration-new.md]: yes

File: docs/tutorials/oss-to-enterprise-migration-same.md (hunk 2 above)
Comment (jplassnibatt) [reply docs/tutorials/oss-to-enterprise-migration-same.md]: yes

**Comment by jplassnibatt** [comment]
> Why are there two files call "-new" and "-same"?

To upgrade in the "same" server that OSS is installed, "new" to use a new server and then use export/import project, I think we should go only with the information from "same", because the "new" should be covered in the export/import process.

File: docs/tutorials/oss-to-enterprise-migration-new.md (hunk 1 above)
Comment (chrismcg14) [reply docs/tutorials/oss-to-enterprise-migration-new.md]: Updated

File: docs/tutorials/oss-to-enterprise-migration-same.md (hunk 2 above)
Comment (chrismcg14) [reply docs/tutorials/oss-to-enterprise-migration-same.md]: Updated

File: docs/tutorials/oss-to-enterprise-migration-new.md (hunk 4 above)
Comment (chrismcg14) [reply docs/tutorials/oss-to-enterprise-migration-new.md:11]: Yes, should I link it in that step?

File: docs/tutorials/oss-to-enterprise-migration-new.md (hunk 5 above)
Comment (chrismcg14) [reply docs/tutorials/oss-to-enterprise-migration-new.md]: Updated

File: docs/tutorials/oss-to-enterprise-migration-new.md (hunk 3 above)
Comment (chrismcg14) [reply docs/tutorials/oss-to-enterprise-migration-new.md]: Updated

**Comment by fdevans** [comment]
Overall I think this could use a reworking on how the content is organized.  There is a lot of repeating content, but some repeats are worded differently than others.  For example Exporting Projects, License install, etc. are all repeated 3 times.  Let's reformat the content to just break out the parts that are install specific and not repeat the others.

**Comment by fdevans** [comment]
Going to close this.  We can revisit the topic as a How To in the learning section later.

**Pull request closed without merging**